biscuit-auth/src/token/authorizer.rs

```diff
@@ -260,8 +260,13 @@ impl<'t> Authorizer<'t> {
             */
     }
 
-    pub fn append(&mut self, other: BlockBuilder) {
-        self.authorizer_block_builder.append(other)
+    pub fn merge(&mut self, mut other: Authorizer) {
+        self.merge_block(other.authorizer_block_builder);
+        self.policies.append(&mut other.policies);
```

Comment:
Thinking about it, a docblock stating that if `other` is already linked to a token, then it'll be discarded should be enough, no need to make `merge` return a `Result`

Reply:
Right! I forgot to ask for clarification about that, as I didn't understand why that would be a problem :/

Reply:
I've added a doc comment to point out that the token is discarded.